src/Common/Models/TaskPlannerInputModel.cs

```diff
@@ -24,5 +25,18 @@ public class TaskPlannerInputModel
 
         [JsonPropertyName("Questions")]
         public List<DialogueModel> Questions { get; set; }
+
+        public bool IsValid()
+        {
+            if ((RobotId == Guid.Empty) | (Id == Guid.Empty))
```

Comment:
It should be `||` instead of `|`. These operations are valid but have different meanings.